src/shared/src/types.rs

```diff
@@ -132,6 +132,7 @@ pub mod user_profile {
     pub struct UserCredential {
         pub credential_type: CredentialType,
         pub verified_date_timestamp: Option<Timestamp>,
+        pub issuer: String,
```

Comment:
Shouldn't this information be a variant?

Reply:
This comes from the config, the `issuer_origin` of `SupportedCredential`. There it's a String, and I believe that's ok.

I don't think we need to create each issuer into an entry in an enum.

Reply:
Analytics might need manual work, and we won't be able to accurately use the information in the frontend for a feature. Those were my spontaneous thoughts for my question, but okay.